src/Controller/UI/AccountController.php

```diff
@@ -440,7 +441,7 @@ public function changePasswordPostAction(Request $request, Ldap $ldap)
         try {
             // Try to add the person to LDAP, incase it needs to re-create.
             $ldap->addPerson($person);
-        } catch (LdapException $exception) {
+        } catch (UidNumberInUseInLDAPException $exception) {
```

Comment:
So, an LdapException happens when the actual update/create method fails. This could happen because the LDAP object didn't exist. Maybe this should just be changed to regular "exception" to it catches both of them?

Reply:
future improvement hopefully